PurchasesTests/TestHelpers/SnapshotTesting+Extensions.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  SnapshotTesting+Extensions.swift
+//
+//  Created by Nacho Soto on 3/4/22.
+
+import Foundation
+import SnapshotTesting
+
+@testable import RevenueCat
+
+// Remove once https://github.com/pointfreeco/swift-snapshot-testing/pull/552 is available in a release.
+extension Snapshotting where Value == Any, Format == String {
```

Comment:
This will go away in a future PR since we will no longer have `[String: Any]` manual encoding.